src/bundle/Resources/views/content/tab/translations/tab.html.twig

```diff
@@ -20,44 +26,57 @@
         <tbody>
         {% for translation in translations %}
             <tr>
-                <td class="ez-table__cell ez-table__cell--has-checkbox">{{ form_widget(form_translation_remove.language_codes[translation.languageCode], {'attr': {'disabled': not translation.canDelete}}) }}</td>
+                <td class="ez-table__cell ez-table__cell--has-checkbox">
+                    {% if can_translate %}
+                        {{ form_widget(form_translation_remove.language_codes[translation.languageCode], {'attr': {'disabled': not translation.canDelete}}) }}
+                    {% endif %}
+                </td>
                 <td class="ez-table__cell">{{ translation.name }}</td>
                 <td class="ez-table__cell">{{ translation.languageCode }}</td>
             </tr>
         {% endfor %}
         </tbody>
     </table>
-    {{ form_end(form_translation_remove) }}
+    {% if can_translate %}
+        {{ form_end(form_translation_remove) }}
+        {% if (is_content_type_translate is defined and is_content_type_translate ) %}
```

Comment:
Why the same template is use for both tabs? It's too bloated for my taste. Make better use of twig blocks and embedding instead of using variables for deciding what to display.